alaoweb/2020.alaoweb.org
# Builds Initial Framework for Poster Sessions

I needed to create a distinct code of conduct layout to facilitate a distinct poster session layout.

To test:
- Site `/posters/` should contain exactly one example poster session
- Clicking on the poster link should display a basic outline for the example poster
- Code of Conduct link in site footer should display without any noticeable changes

This is an initial framework. If it looks adequate as a starting point, let's close #31 and create smaller issues for enhanced functionality and UX improvements.
